java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/Server.java

```diff
@@ -123,6 +123,7 @@ public static boolean isClientResponseThread(NbCodeLanguageClient client) {
     }
     
     public static NbLspServer launchServer(Pair<InputStream, OutputStream> io, LspSession session) {
+        System.setProperty("org.netbeans.modules.java.source.usages.BinaryAnalyser.fullIndex", "true");
```

Comment:
I am not sure what is this, but if we want to configure behavior of some other module, we should do it via branding API, not system property.

Reply:
Solved by providing `BinaryAnalyser.Config` implemementation in VSNetBeans integration module.

Reply:
Functionally better, but the implementation dependency!